packages/app/obojobo-express/server/migrations/20230301155626-create-notification.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
```

Comment:
I would probably regenerate these migrations so their filenames indicate a more recent date, to make sure they're run in the correct order.